docs/MVS_Outputs.rst

```diff
@@ -279,6 +279,22 @@ As above, we apply a weighting based on Electricity Equivalent.
        Degree of Autonomy = \frac{\sum_{i} E_{demand,i} \cdot w_i - \sum_{j} E_{consumption,provider,j} \cdot w_j}{\sum_{i} E_{demand,i} \cdot w_i}
 
 
+Degree of net zero energy (NZE)
+###############################
+
+The degree of net zero energy describes the overall renewable energy generated locally minus the excess energy divided by the overall energy consumed.
+In NZE systems import and export of energy is allowed while the balance over one year should be zero. If more energy is exported than imported it is plus-energy system.
```

Comment:
```suggestion
The degree of net zero energy describes the ability of an energy system to provide its aggregated annual demand though renewable, local sources. For that, the balance between local generation as well as consumption from and feed-in towards the energy provider is compared. In a net zero energy system, demand can be supplied by energy import, but then local renewable energy generation should allow an equally high energy export of energy in the course of the year. In a plus energy system, the export exceeds the import, while local renewable generation can supply all demand (from an aggregated perspective).
To calculate the degree of NZE, the overall renewable energy generated locally minus the excess energy is divided by the overall energy consumed.
```

Reply:
Thank you this is a good explanation I could not bring to "paper" that day.